staging/src/github.com/kubeedge/mapper-framework/pkg/config/config.go

```diff
@@ -51,13 +46,8 @@ type Common struct {
 	EdgeCoreSock string `yaml:"edgecore_sock"`
 }
 
-type DevInit struct {
-	Mode      string `yaml:"mode"`
-	Configmap string `yaml:"configmap"`
-}
-
 // Parse the configuration file. If failed, return error.
-func (c *Config) Parse() error {
+func Parse(c *Config) error {
```

Comment:
1. The flags parse is best extracted by `cobra.Command`, like `cmd/cloudcore/app/server.go`;
2. The `Parse(..)` can input a config file path to initialize global config variables;
    ```golang
    func Parse(configFile string) error {}
    ```
3. The zero value of a pointer is nil
    ```golang
    var c *config.Config
    config.Parse(c)

    // config.go
    func Parse(c *config.Config) error {
        // c is nil
    }
    ```

Reply:
ok, I'll fix it.